nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/resources/json/output/dataTypes.json

```diff
@@ -11,6 +11,7 @@
   "time" : "17:00:00",
   "timestamp" : "2017-01-01 17:00:00",
   "char" : "c",
+  "decimal" : "45.76",
```

Comment:
I would rather have this as a number to be honest. There is a specific case in the `WriteJsonResult` class.